### PR TITLE
[OSDOCS-8735][OSD] 'OSD Service Def > Logging' generalizes use of Amazon CloudWatch

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.14+
Issue:

https://issues.redhat.com/browse/OSDOCS-8735
Link to docs preview:

https://68125--docspreview.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition#sdpolicy-logging_osd-service-definition
image

https://68125--docspreview.netlify.app/openshift-dedicated/latest/logging/log_collection_forwarding/log-forwarding#logging-vector-fluentd-feature-comparison_log-forwarding
image

https://68125--docspreview.netlify.app/openshift-dedicated/latest/logging/log_collection_forwarding/log-forwarding#log-forwarding-collector-outputs_log-forwarding
image

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jaybeeunix** [comment]
all of these changes look good to me :+1:

**Comment by mletalie** [comment]
Hello @xueli181114 , can I please get a QE review on this PR? Thanks!

**Comment by anpingli** [comment]
LGTM

File: modules/sdpolicy-logging.adoc [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
 == Cluster audit logging
-Cluster audit logs are available through Amazon CloudWatch, if the integration is enabled. If the integration is not enabled, you can request the audit logs by opening a support case. Audit log requests must specify a date and time range not to exceed 21 days. When requesting audit logs, customers should be aware that audit logs are many GB per day in size.
-
-
+Cluster audit logs are available through Amazon CloudWatch (on AWS) or Google Cloud Logging (on GCP), if the integration is enabled. If the integration is not enabled, you can request the audit logs by opening a support case. Audit log requests must specify a date and time range not to exceed 21 days. When requesting audit logs, customers should be aware that audit logs are many GB per day in size.
 [id="application-logging_{context}"]
 == Application logging
-Application logs sent to `STDOUT` are collected by Fluentd and forwarded to Amazon CloudWatch through the cluster logging stack, if it is installed.
+Application logs sent to `STDOUT` are forwarded to Amazon CloudWatch (on AWS) or Google Cloud Logging (on GCP) through the cluster logging stack, if it is installed.
```
Comment (JoeAldinger) [inline modules/sdpolicy-logging.adoc:16]: I would add a noun after `STDOUT` for clarity. Maybe file?

**Review by JoeAldinger — COMMENTED** [review]
Small suggestion, otherwise lgtm.

**Pull request merged**